app/views/application/_likes.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_for(@post.likes.new, url: user_post_likes_path(@post.author, @post)) do |form| %>
```

Comment:
- [ ] I can't display the views as this method throws an error. Kindly fix this.
![image](https://github.com/Izebiafe/Blog/assets/91486791/6daf0e2b-a4fe-4577-ad0e-58f5fec83705)